tests/queries/tests.py

```diff
@@ -2079,6 +2083,18 @@ def test_distinct_ordered_sliced_subquery(self):
             ).order_by('id').values_list('id', flat=True), [2, 4]
         )
 
+    def test_logical_xor_subquery(self):
+        """Subquery should support logical XOR on supported databases"""
```

Comment:
This docstring doesn't have much value, I'd remove it.

Reply:
I'm going to push back on this one a little:

I find it best to ALWAYS have a docstring explaining what the test is trying to test. This way the name and code don't need to be deciphered, and there is no confusion about the intended goal of the test.

Further, I see other tests throughout here with similar short docstrings?

Reply:
We have many old tests in `queries` with unnecessary docstrings.

Reply:
I'm sorry, I just can't find it in me to make my tests less readable. I know you said you had some nitpicks you wanted to change before merging, perhaps you can do that?

Or can you suggest a more descriptive docstring for the test that explains what its testing?